sym/manifest.json

```
@@ -0,0 +1,64 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "d81d1dd3-d5e8-4373-98a6-f95b797b3f9d",
+    "app_id": "sym",
+    "display_on_public_website": true,
+    "tile": {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Send Sym Audit Logs to Datadog",
+        "title": "Sym",
+        "media": [
+            {
+                "media_type": "video",
+                "caption": "A Sym Overview Video",
+                "image_url": "images/sym_video_thumbnail.jpg",
+                "vimeo_id": 123456789
+            },
+            {
+                "media_type": "image",
+                "caption": "Sym helps you build access and approval workflows using your native platform engineering tools",
+                "image_url": "images/home_hero_image.png"
+            },
+            {
+                "media_type": "image",
+                "caption": "Define and deploy access rules in Terraform",
+                "image_url": "images/define_deploy.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "Request, approve, and deny anywhere in Slack",
+                "image_url": "images/request_approve.jpg"
+            }
+        ],
+        "classifier_tags": [
+            "Submitted Data Type::Logs"
```

```suggestion
        "classifier_tags": [
            "Supported OS::Windows",
            "Supported OS::macOS",
            "Supported OS::Linux",
            "Category::Security",
            "Offering::Integration",
            "Submitted Data Type::Logs"
```
Added a few tags I thought were applicable.